explorer/client/src/utils/static/mock_data.json

```diff
@@ -443,6 +443,58 @@
                     }
                 }
             }
+        },
+        {
+            "hI1m16cQkwErCDqbPyuV2ZtX4DZEel2lQTU1b1VJq0c=": {
```

Comment:
For this transaction item we have no `id` field. This means that the `findDataFromID` function in `src/utils/static/searchUtil.ts` will need to be updated to handle both this unusual case as well as addresses and objects, which do have an `id` field in the JSON data items delivered.